app/graphql/types/query_type.rb

```diff
@@ -52,10 +53,24 @@
         return GraphQL::ExecutionError.new("Invalid user token. Please log in.")
       end
       results = PgSearch.multisearch(args["query"])
+      results = results.select{ |r| r.searchable_type == 'Article'}
       results.select{ |r| !r.nil? && !(r.searchable.is_published) }
     }
   end
   
+  field :searchUsers do
+    type !types[Types::UserType]
+    argument :query, !types.String
+    description "Find users by slug"
+    resolve -> (obj, args, ctx) {
+      if !Authentication::editor_is_valid(ctx)
+        return GraphQL::ExecutionError.new("Invalid user token. Please log in.")
+      end
+      results = PgSearch.multisearch(args["query"])
```

Comment:
Slightly confused by this: aren't you supposed to edit query into slug form atm? like replace " " with "-" .downcase